tcms/core/tests/test_forms.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+
+from tcms.core.forms.fields import UserField
+from tcms.tests.factories import UserFactory
+
+
+class TestUserField(TestCase):
```

Comment:
This entire test case can inherit directly from unittest.TestCase instead of this Django class. 

The UserField class doesn't need a database to perform clean so all of this can be sped up if you inherit from unittest.TestCase and just pass the valid/invalid values directly to `.clean()`

Reply:
Ok, but `clean` returns user object from DB, which needs to be asserted, i am not sure if its sufficient to pass valid/invalid values and assert the exceptions.  I see that, these tests seem somewhere between unit and component level, but IMO a test user is needed on DB.

https://github.com/kiwitcms/Kiwi/blob/4fd940a7d2e1c6410158684cc83068f3922ead49/tcms/core/forms/fields.py#L28

Reply:
Omg, you are right. I was only experimenting with the negative scenarios which don't need the dabatase, but the positive ones do need it. My bad.